experiment/autobumper/bumper/bumper.go

```diff
@@ -206,9 +234,7 @@ func Run(o *Options) error {
 		return fmt.Errorf("failed to change to root dir: %w", err)
 	}
 
-	images, err := UpdateReferences(
-		o.BumpProwImages, o.BumpBoskosImages, o.BumpTestImages, o.TargetVersion,
-		o.IncludedConfigPaths, o.ExcludedConfigPaths, o.ExtraFiles)
+	images, err := UpdateReferences(o)
```

Comment:
This now takes an options struct with 15 parameters, most of which used to be constants, that has no defaulting. Can we in the future please be a bit thoughtful about backwards compatibility when changing exported interfaces?